packages/@aws-cdk/cx-api/lib/cxapi.ts

```diff
@@ -24,10 +24,17 @@ export type CXRequest = ListStacksRequest | SynthesizeRequest;
  * Represents a missing piece of context.
  * (should have been an interface, but jsii still doesn't have support for structs).
  */
-export interface MissingContext {
+export interface ContextProviderProps {
```

Comment:
after reading this again (a few days later). I think I'm ready to switch this back to `MissingContext`.

```ts
export interface MissingContext {
  provider: string;
  props: {
    account: string;
    region: string;
    [key: string]: string;
  };
}
```